docs/source/index.rst

```diff
@@ -82,6 +82,12 @@ Version 2.0
 - Changed the behavior of `combine_slices` to stack slices from head (slice 0)
   to foot (slice -1). Note that this is the reverse of the behavior in v1.*.
 
+Version 3.0
```

Comment:
👍

Reply:
I didn't notice this at first, but should this be "0.3.0"? I am not sure why I used "1.5" and "2.0" in the previous documentation entries ... 

Looking at https://pypi.org/project/dicom-numpy/#history

It seems like these should be "0.1.5" and "0.2.0" as well